config/features.rb

```diff
@@ -19,4 +19,8 @@
   feature :import_mode,
           default: false,
           description: "Import mode - Background jobs are run on specially named queues"
+
```

Comment:
I am not sure what this `feature` is for, but this is going to become an integral part of our repository setup and will be enabled for every repository we run -- ever. So won't ever need to disable it.
As far as I am concerned, this can be removed :)

Reply:
@fradeve I was thinking more of other people using the HA codebase that didn't have this setup in mind. as this feature is very specific to Ubiquity.

Reply:
:+1: This should get pulled out to the UP specific engine whenever we make that or it could get pulled out into its own specific little gem.

Reply:
I was thinking the same thing. It would be best in its own Gem probably, but i'll carry on developing inside HA for now.